FV3/makefile

```diff
@@ -50,7 +46,7 @@ libs:
 	$(MAKE) -C atmos_cubed_sphere    $(MAKE_OPTS) FMS_DIR=$(FMS_DIR)
 	$(MAKE) -C ../stochastic_physics $(MAKE_OPTS) FMS_DIR=$(FMS_DIR) 32BIT=N  # force gfs physics to 64bit
 
-libs_no_dycore:
+libs_no_dycore: fms_check
```

Comment:
I had to refactor the previous hard-check of FMS to be a make requirement, so that it wouldn't fail when running the serialize_preprocess target without FMS present.